_plugins/asset_path_tag.rb

```diff
@@ -0,0 +1,129 @@
+<<~aid*- coding: utf-8 -*- ruby -*- _plugins/asset_path_tag.rb *****************
+
+  A Jekyll plugin to output a relative URL for assets based on the post or page
+
+  MIT License copyright original author(s); minimally adapted for passing strict
```

Comment:
```suggestion
  MIT License copyright respective author(s); minimally adapted for passing strict
```

hang on, there is a formal of doing it nicely; lemme check

Reply:
🔗 https://rubydoc.info/gems/yard/file/docs/Tags.md#author

Reply:
this is a freaking rabbit hole; beware!